packages/iov-tendermint-rpc/src/common.ts

```diff
@@ -33,7 +33,7 @@ export interface JsonRpcEvent {
   readonly query: string;
   readonly data: {
     readonly type: string;
-    readonly value: {};
+    readonly value: any;
```

Comment:
Question: do you need this?

`{}` kind of "enforces" that value is not a number or string. `any` allows those as well. If we have those types, please do change this. Otherwise, I don't see it adds anything

Reply:
The problem is that with an interface definition like this, every user needs to explicitly cast to any.

Before:

`(events[i].data.value as any).header.height`

After:

`events[i].data.value.header.height`

The old version has no better type safety in the getter but is far harder to read. I see that this allows assigning numbers and strings, but as long as there is no "any interface" type, I think `any` describes the property best.